contracts/governor/src/errors.rs

```diff
@@ -14,4 +15,12 @@ pub enum GovernorError {
     AllowanceError = 9,
     BalanceError = 10,
     OverflowError = 12,
+
+    NonExistentProposalError = 13,
+    ProposalNotActiveError = 14,
+    InvalidProposalSupportError = 15,
+    VotePeriodNotFinishedError = 16,
+    ProposalNotQueuedError = 17,
+    TimelockNotMetError = 18,
+    CancelActiveProposalError = 19,
```

Comment:
these numbers overlap built-in contract error reporting. Can we use 200 to start?

IE - NonExistentProposalError = 200, and so on.

Votes is starting at 100